typescript/ws-protocol/README.md

```diff
@@ -99,28 +88,11 @@ async function main() {
 main().catch(console.error);
 ```
 
-## Example client
+## Client template
 
-The following script provides a simple example client which subscribes to messages on all channels that use the `json` encoding. To get started with the example client:
+The example client template below subscribes to messages on all channels that use the `json` encoding. 
 
-1. Copy the script below into a file named `client.js`.
-2. Run the [example server above](#example-server) or any other Foxglove WebSocket server.
-3. In a separate terminal, run `node client.js`.
-
-```
-$ node client.js
-{
-  subscriptionId: 0,
-  timestamp: 1638999307183000000n,
-  data: { msg: 'Hello!', count: 2849 }
-}
-{
-  subscriptionId: 0,
-  timestamp: 1638999307384000000n,
-  data: { msg: 'Hello!', count: 2850 }
-}
-...
-```
```

Comment:
You don't think it's helpful to show expected output if the client is working properly?